### DISCUSSION TITLE
Task Manager similar to Chrome

### DISCUSSION THREAD

**Discussion by Daszbin** [cefsharp/CefSharp#4306] [General]
So all chromium based browser have task manager. Is it possible currently in Cefsharp 107 to have this.
![Screenshot (34)](https://user-images.githubusercontent.com/37669537/202071284-ca17653d-e1e9-4162-9cff-151776e3d1b7.png)

**Comment by amaitland** [comment]
No, CEF doesn't provide a task manager.

**Reply by Daszbin** [reply]
is their anyway we could implement something similar since we have developer tools and all chromium browsers have this.

**Reply by amaitland** [reply]
Not directly. CEF would need to add support first. 

Read https://github.com/cefsharp/CefSharp/blob/master/.github/ISSUE_TEMPLATE/feature_request.md for background.
